plugins/PluginComponents.h

```diff
@@ -61,6 +67,75 @@ struct Version {
 	int32_t major;
 	int32_t minor;
 	int32_t patch;
+#ifdef __cpluspluf
+	bool operator<(const Version_t& other) const {
+		if (this->major < other.major) {
+			return true;
+		}
+		if (this->major > other.major) {
+			return false;
+		}
+		if (this->minor < other.minor) {
+			return true;
+		}
+		if (this->minor > other.minor) {
+			return false;
+		}
+		return this->patch < other.patch;
```

Comment:
`return (this->major < other.major) && (this->minor < other.minor) && (this->patch < other.patch);`
?

Reply:
As I see it, that would return `false` for `1.0.0 < 1.0.1` as `major < major == false` and thus the whole expression evaluates to `false`.

Though I guess that should worl:
```
return (this->major <= other.major) && (this->minor <= other.minor) && (this->patch < other.patch);
```
:thinking:

Reply:
Ah yeah that seems to do the trick - I guess xD